### PR TITLE
Create script to check for unimplemented exercises and submit issue

### PR DESCRIPTION
**Pull request by lemoncurry**
This script checks for unimplemented exercises and provides the possibility to submit a new issue to the track. A generic text for issue submission is provided.

Also a description on how to use the script is added to the CONTRIBUTING.md file.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -222,3 +222,29 @@ To run this script:
   4. Run `./scripts/create_issues_versionchange_canonical.sh -t . -s --spec-path path_to_problem_specifications` from the root of this repository and follow the directions.
   
   5. If you submitted new issues, please check these submissions on the [issues page](https://github.com/exercism/java/issues).
+
+## Checking exercises are implemented and submit new issues
+
+There is [a script which allows you to submit new issues](https://github.com/exercism/java/blob/master/scripts/create_issues_new_exercise.sh) to this repo with generic title, description and labels if am unimplemented exercise is detected.
```
Comment (FridaTveit) [inline CONTRIBUTING.md]: "if an unimplemented execise" I think you mean 🙂

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -222,3 +222,29 @@ To run this script:
...
+To run this script:
+
+  1. Clone [the problem-specifications repository](https://github.com/exercism/problem-specifications).
+  
+  2. Create a file `.exercism-version-update-issue-script-settings.sh` in your home directory.
+  
+  3. In this file, you have to put the following variables:
+        - `TOKEN="your_token"`
+        - `OWNER="repo_owner"` # -> `"exercism"`
+        - `REPO="repo_name"` # -> `"java"`
```
Comment (FridaTveit) [inline CONTRIBUTING.md]: I would probably just write this as:
```
TOKEN="your_token"
OWNER="exercism"
REPO="java"
```
Just to make it 100% clear what needs to be in the file 🙂

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -222,3 +222,29 @@ To run this script:
...
+        
+     For authentication, you need to create a personal token, see [this GitHub page](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) for more information.
+
+  4. Run `./scripts/create_issues_new_exercise.sh -t . -s --spec-path path_to_problem_specifications` from the root of this repository and follow the directions.
+  
+  5. If you submitted new issues, please check these submissions on the [issues page](https://github.com/exercism/java/issues).
```
Comment (FridaTveit) [inline CONTRIBUTING.md]: I think I would reword this sentence a bit, to make it a bit more clear. Maybe something like "If you do submit a new issue you can find the opened issue on the issues page". What do you think? 🙂

File: scripts/create_issues_new_exercise.sh [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+        title="$exercise: implement exercise"
+        body="The exercise **$exercise** has not been implemented yet for the Java track. \nThe description of the exercise can be found in the [problem specification repository](https://github.com/exercism/problem-specifications/tree/master/exercises/$exercise). \n\nHow to implement a new exercise for the Java track is described in detail in [CONTRIBUTING.md](https://github.com/exercism/java/blob/master/CONTRIBUTING.md#adding-a-new-exercise). Please have a look there first before starting working on the exercise. \nAlso please make sure it is clear that you are currently working on this issue, either by asking to be assigned to it, or by opening an empty PR. \n\nWhen opening an PR, please reference this issue using any of the [closing keywords](https://help.github.com/en/articles/closing-issues-using-keywords). \n\nIf you had a look at the exercise description and you concluded that the exercise might not be possible to implement in the Java language, please leave a comment and describe the problem. \n\nIn case you have any further questions, feel free to ask here. \n"
```
Comment (FridaTveit) [inline scripts/create_issues_new_exercise.sh]: I think I would change "If you had a look" to "If you have had a look", just to be more clear 🙂

File: CONTRIBUTING.md (hunk 1 above)
Comment (FridaTveit) [inline CONTRIBUTING.md]: I think this first sentence doesn't quite make it clear enough what the script is for, it kind of sounds like it's for issues in general, whereas it's meant to be for opening issues for unimplemented exercises I believe? If that's the case then it might be worth rewording it to something like:
"There is a script which allows you to easily check if there are any exercism exercises which haven't been implemented in the Java track, and create issues for those exercises if there are any."
Not necessarily exactly that, but something that makes it clear that this is for unimplemented exercises, if that makes sense? 🙂

**Comment by lemoncurry** [comment]
Thank you @FridaTveit!
